src/topology/algebra/infinite_sum.lean

```diff
@@ -56,7 +56,8 @@ def summable (f : β → α) : Prop := ∃a, has_sum f a
 /-- `∑' i, f i` is the sum of `f` it exists, or 0 otherwise -/
 @[irreducible] def tsum {β} (f : β → α) := if h : summable f then classical.some h else 0
 
-notation `∑'` binders `, ` r:(scoped f, tsum f) := r
+-- See the comment on precedence of big operators in `algebra/big_operators/basic.lean`.
```

Comment:
Is it worth upgrading that comment to a library note?

Otherwise LGTM.
bors d+